boilerplate/openshift/golang-osd-operator/build-opm-catalog.sh

```diff
@@ -0,0 +1,379 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Global vars
+REPO_ROOT=$(git rev-parse --show-toplevel)
+
+## Global vars to modify behaviour of the script
+DRY_RUN=${DRY_RUN:-false}
+DELETE_TEMP_DIR=${DELETE_TEMP_DIR:-true}
+BRANCH=${BRANCH:-master}
+SET_X=${SET_X:-false}
+NO_LOG=${NO_LOG:-false}
+[[ "$SET_X" != "false" ]] && set -x
+
+function log() {
+    local to_log=${1}
+    [[ "$NO_LOG" != "false" ]] && return 0
+    local msg
+    msg="$(date "+%Y-%m-%d %H:%M:%S")"
+    [[ "$DRY_RUN" != "false" ]] && msg="$msg -- DRY-RUN"
+    echo "$msg -- $to_log" 1>&2
+}
+
+function check_required_environment() {
+    local count=0
+    for var in OLM_BUNDLE_IMAGE \
+               OLM_CATALOG_IMAGE \
+               OPERATOR_IMAGE \
+               OPERATOR_IMAGE_TAG \
+               OPERATOR_VERSION \
+               OPERATOR_NAME \
+               CONTAINER_ENGINE \
+               CONTAINER_ENGINE_CONFIG_DIR \
+               CURRENT_COMMIT \
+               OLM_CHANNEL \
+               OLM_BUNDLE_VERSIONS_REPO
```

Comment:
Since the bundle versions repo is so tightly bound to other variables that aren't "parameters" to this script (e.g. the path to the versions file), suggest simply hardcoding it and getting rid of this variable. (IMO the more we can reduce the number of env vars we're relying on, the better.)

Reply:
I was leaving this as a parameter because it's helpful to be able to test against a different git repository if needed. But I can remove it and make it overridable if needed to remove it from the Makefiles as it is just testing convenient.

Reply:
Fixed in 1bd829f